Resources/frontend/js/jquery.adyen-confirm-order.js

```diff
@@ -176,17 +177,19 @@
                             },
                         });
                     },
-                onError: function (error) {
-                    console.log(error);
-                }
+                    onError: function (error) {
+                        console.log(error);
+                    }
                 })
                 .mount('#AdyenChallengeShopperThreeDS2');
         },
 
         handlePaymentDataRedirectShopper: function (data) {
-            var me = this;
-            if (data.action.type === 'redirect') {
-                me.adyenCheckout.createFromAction(data.action).mount(me.opts.mountRedirectSelector);
+            const me = this;
+            if ('redirect' === data.action.type || 'qrCode' === data.action.type) {
```

Comment:
For now this works but for later it's maybe easier to check the resultCode instead of the type. All the RedirectShopper and Pending resultCodes should have an action which needs to be shown in the checkout starting with the adyen checkout api version 64.